ocdata/configs/constants.py

```diff
@@ -0,0 +1,697 @@
+    92: "U",
+    93: "Np",
+    94: "Pu",
+    95: "Am",
+    96: "Cm",
+    97: "Bk",
+    98: "Cf",
+    99: "Es",
+    100: "Fm",
+    101: "Md",
+    102: "No",
+    103: "Lr",
+    104: "Rf",
+    105: "Db",
+    106: "Sg",
+    107: "Bh",
+    108: "Hs",
+    109: "Mt",
+    110: "Ds",
+    111: "Rg",
+    112: "Cn",
+    113: "Nh",
+    114: "Fl",
+    115: "Mc",
+    116: "Lv",
+    117: "Ts",
+    118: "Og",
+}
+
+# Covalent radius of elements (unit is pm, 1pm=0.01 angstrom)
+# Value are taken from https://github.com/lmmentel/mendeleev
+COVALENT_RADIUS = {
+    "H": 32.0,
+    "He": 46.0,
+    "O": 63.0,
+    "F": 64.0,
+    "Ne": 67.0,
+    "N": 71.0,
+    "C": 75.0,
+    "B": 85.0,
+    "Ar": 96.0,
+    "Cl": 99.0,
+    "Be": 102.0,
+    "S": 103.0,
+    "Ni": 110.0,
+    "P": 111.0,
+    "Co": 111.0,
+    "Cu": 112.0,
+    "Br": 114.0,
+    "Si": 116.0,
+    "Fe": 116.0,
+    "Se": 116.0,
+    "Kr": 117.0,
+    "Zn": 118.0,
+    "Mn": 119.0,
+    "Pd": 120.0,
+    "Ge": 121.0,
+    "As": 121.0,
+    "Rg": 121.0,
+    "Cr": 122.0,
+    "Ir": 122.0,
+    "Cn": 122.0,
+    "Pt": 123.0,
+    "Ga": 124.0,
+    "Au": 124.0,
+    "Ru": 125.0,
+    "Rh": 125.0,
+    "Al": 126.0,
+    "Tc": 128.0,
+    "Ag": 128.0,
+    "Ds": 128.0,
+    "Os": 129.0,
+    "Mt": 129.0,
+    "Xe": 131.0,
+    "Re": 131.0,
+    "Li": 133.0,
+    "I": 133.0,
+    "Hg": 133.0,
+    "V": 134.0,
+    "Hs": 134.0,
+    "Ti": 136.0,
+    "Cd": 136.0,
+    "Te": 136.0,
+    "Nh": 136.0,
+    "W": 137.0,
+    "Mo": 138.0,
+    "Mg": 139.0,
+    "Sn": 140.0,
+    "Sb": 140.0,
+    "Bh": 141.0,
+    "In": 142.0,
+    "Rn": 142.0,
+    "Sg": 143.0,
+    "Fl": 143.0,
+    "Tl": 144.0,
+    "Pb": 144.0,
+    "Po": 145.0,
+    "Ta": 146.0,
+    "Nb": 147.0,
+    "At": 147.0,
+    "Sc": 148.0,
+    "Db": 149.0,
+    "Bi": 151.0,
+    "Hf": 152.0,
+    "Zr": 154.0,
+    "Na": 155.0,
+    "Rf": 157.0,
+    "Og": 157.0,
+    "Lr": 161.0,
+    "Lu": 162.0,
+    "Mc": 162.0,
+    "Y": 163.0,
+    "Ce": 163.0,
+    "Tm": 164.0,
+    "Er": 165.0,
+    "Es": 165.0,
+    "Ts": 165.0,
+    "Ho": 166.0,
+    "Am": 166.0,
+    "Cm": 166.0,
+    "Dy": 167.0,
+    "Fm": 167.0,
+    "Eu": 168.0,
+    "Tb": 168.0,
+    "Bk": 168.0,
+    "Cf": 168.0,
+    "Gd": 169.0,
+    "Pa": 169.0,
+    "Yb": 170.0,
+    "U": 170.0,
+    "Ca": 171.0,
+    "Np": 171.0,
+    "Sm": 172.0,
+    "Pu": 172.0,
+    "Pm": 173.0,
+    "Md": 173.0,
+    "Nd": 174.0,
+    "Th": 175.0,
+    "Lv": 175.0,
+    "Pr": 176.0,
+    "No": 176.0,
+    "La": 180.0,
+    "Sr": 185.0,
+    "Ac": 186.0,
+    "K": 196.0,
+    "Ba": 196.0,
+    "Ra": 201.0,
+    "Rb": 210.0,
+    "Fr": 223.0,
+    "Cs": 232.0,
+}
+
+# We will enumerate surfaces with Miller indices <= MAX_MILLER
+MAX_MILLER = 2
+
+# We will create surfaces that are at least MIN_XY Angstroms wide. GASpy uses
+# 4.5, but our larger adsorbates here can be up to 3.6 Angstroms long. So 4.5 +
+# 3.6 ~= 8 Angstroms
+MIN_XY = 8.0
+
+COVALENT_MATERIALS_MPIDS = [
```

Comment:
maybe we should write something to determine this? As we add materials to the bulk db we have to remember not to neglect this.

Reply:
resolving because muhammed started a thread too